.github/workflows/treetracker-wallet-api-test-e2e.yml

```diff
@@ -0,0 +1,28 @@
+name: Tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    env:
```

Comment:
I suggest changing to use instant DB just like what we are using in the current CI, or we can directly use that one. @ZavenArra right?